phpBB/language/en/acp/attachments.php

```diff
@@ -37,7 +37,7 @@
 // in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
 
 $lang = array_merge($lang, array(
-	'ACP_ATTACHMENT_SETTINGS_EXPLAIN'	=> 'Here you can configure the main settings for attachments and the associated special categories.',
+	'ACP_ATTACHMENT_SETTINGS_EXPLAIN'	=> 'Here you can configure the main settings for attachments. The associated special categories settings that control which file types may be attached are found on the "Postings" tab under "Attachments".',
```

Comment:
After further looking at this, I do have to agree with @VSEphpbb. This does not help much but it something we should worry about while restructuring the ACP: